pkg/cli/args.go

```diff
@@ -0,0 +1,67 @@
+package cli
+
+import (
+	libflag "flag"
+	"io"
+	"strings"
+
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	OneDashFlag    NormalizeActsType = iota
+	DoubleDashFlag NormalizeActsType = iota
+)
+
+type NormalizeActsType byte
+
+type Args struct {
```

Comment:
Same here... Explain we're wrapping the `urfave` package of the same name, and what we're adding.

Reply:
Looks like we wrap a bunch of other packages below too... Same comment applies to them.

Reply:
Agree, added unit tests and comments.